app/controllers/users_controller.rb

```diff
@@ -10,13 +10,36 @@ def create
     user = User.new(user_params)
     if user.save
       session[:user_id] = user.id
+
+      # send confirmation email with token to register
```

Comment:
do we want to leave this comment here for the time being?

Reply:
Yeah, I just had it there so we could keep track of what was happening. We will go through all of our code for this kind of thing later I think. Or just get it on the next push